rfc_frontmatter.md

```diff
@@ -0,0 +1,15 @@
+% Title = "FF Video Codec 1"
+% abbrev = "FFV1"
+% category = "info"
+% docName = ""
```

Comment:
You should give some input, e.g.:
`docName = "draft-niedermayer-cellar-ebml-00"`